### ISSUE TITLE
Oracle JDK 9 Issue

### ISSUE THREAD

**Issue by allan-shoup** [sviperll/ozymandias#7]
When the JDK running maven is Oracle's JDK 9, I get `java.lang.ClassNotFoundException: javax.xml.bind.JAXBException` when trying to execute *coreext-maven-plugin*. If I run with OpenJDK 9, this error does not occur.

The full log is attached.
[mvn.log](https://github.com/sviperll/ozymandias/files/1861226/mvn.log)

**Comment by allan-shoup** [comment]
As per [this Stack Overflow answer](https://stackoverflow.com/a/43574427), this can be worked around by adding the following dependencies to the plugin
```
                <dependencies>
                    <dependency>
                        <groupId>javax.xml.bind</groupId>
                        <artifactId>jaxb-api</artifactId>
                        <version>2.2.11</version>
                    </dependency>
                    <dependency>
                        <groupId>com.sun.xml.bind</groupId>
                        <artifactId>jaxb-core</artifactId>
                        <version>2.2.11</version>
                    </dependency>
                    <dependency>
                        <groupId>com.sun.xml.bind</groupId>
                        <artifactId>jaxb-impl</artifactId>
                        <version>2.2.11</version>
                    </dependency>
                </dependencies>
```

**Comment by sviperll** [comment]
Thank you very much for your report.

May be you can provide PR for it? I'll try to make new release before monday

**Comment by hanson76** [comment]
Pullrequest #8 should fix JDK 9+ compability

**Comment by hanson76** [comment]
THanks for merging my pull request, is it possible to get a release with this change?

**Comment by sviperll** [comment]
Year, sorry, I've tried it, but failed because of local maven repository configuration :(
I'll try to do it this evening.

**Comment by sviperll** [comment]
Released `0.23`, thank you very much for your contribution!:)
